package-lock.json

```diff
@@ -17,7 +17,8 @@
         "gifenc": "^1.0.3",
         "libtess": "^1.2.2",
         "omggif": "^1.0.10",
-        "opentype.js": "^1.3.1"
+        "opentype.js": "^1.3.1",
```

Comment:
oh this reminds me, we can probably `npm uninstall opentype.js` too if we no longer use it